src/Components/Cards.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Cards = props => {
+
+    return (
+
```

Comment:
I see you made a cards component, but I'm not seeing it used anywhere?  Did you have some issues implementing it?